docs/installation/introduction.rst

```diff
@@ -18,62 +17,61 @@ the following steps:
 #. :doc:`Post installation <./post-install>` configuration steps and sanity
    checks
 
+.. warning::
+
+   MetalK8s is not designed to handle world-distributed multi-site
+   architectures. Instead, it provides a highly resilient cluster at the
+   datacenter scale. To manage multiple sites, look into application-level
+   solutions or alternatives from such the communities as `the Multicluster SIG
```

Comment:
> from such the communities as

That's weird to read. And a SIG isn't a community itself, it's just a group within Kubernetes community. I'd rather have this sentence read: "or alternatives from the community (e.g. in the scope of the Multicluster SIG)".

Reply:
uh... yeah. That "the" has no business in there. That's an error, so good catch. Rewording.